elyra/pipeline/airflow/processor_airflow.py

```diff
@@ -280,12 +280,24 @@ def _cc_pipeline(self, pipeline, pipeline_name):
                         continue
 
                     # Get corresponding property's value from parsed pipeline
-                    property_value = operation.component_params.get(component_property.ref)
+                    property_value_dict = operation.component_params.get(component_property.ref)
+                    property_value = property_value_dict[property_value_dict['activeControl']]
+
+                    self.log.debug(f"Property value : {property_value}")
```

Comment:
I find this in need of some comments.  Reverse engineering this a bit, `property_value_dict` is a dictionary of name/value pairs where the name (key) is a property name.  In the case of the "activeControl" property, its value is another property name.  Its value is, I'm assuming, the name of the control chosen by the user based on the behavior associated with the "OneOfConrol".

I think some kind of comment explaining this would be helpful along with an update to the debug statement to include the "discovered" property name.  Sans the comment, perhaps something like this:
```suggestion
                    property_value_dict = operation.component_params.get(component_property.ref)
                    property_name = property_value_dict['activeControl']
                    property_value = property_value_dict[property_name]

                    self.log.debug(f"Property name : {property_name}, value : {property_value}")
```

Reply:
updated with changes